examples/06-verif-manual/vm-009-large_lateral_deflection_of_unequal_stiffness_springs.py

```diff
@@ -0,0 +1,409 @@
+# Post-processing
+# ~~~~~~~~~~~~~~~
+# Enter post-processing, avoiding the printing output.
+
+# Enter the post-processing mode.
+_ = mapdl.post1()
+
+
+###############################################################################
+# Getting Displacements
+# ~~~~~~~~~~~~~~~~~~~~~
+# Enter post-processing. To get results of the strain energy and displacements
+# in X and Y directions from the node where the force is applied using
+# :meth:`Mapdl.get_value <ansys.mapdl.core.Mapdl.get_value>`.
+
+# Defines the data set to be read from the results file by the time-point.
+mapdl.set(time=15)
+
+# Fills a table of element values for further processing for strain energy.
+mapdl.etable("SENE", "SENE")
+
+# Sum all active entries in element stress table.
+mapdl.ssum()
+
+# Get the value of the stain energy of the spring elements.
+strain_energy = mapdl.get_value(entity="SSUM",
+                                entnum=0,
+                                item1="ITEM",
+                                it1num="SENE")
+
+# Prints nodal solution results of the X, Y, and Z structural displacements
+# and vector sum.
+print(mapdl.prnsol("U", "COMP"))
+
+# Get the value of the displacements in X-direction.
+disp_x = mapdl.get_value(entity="NODE",
+                         entnum=2,
+                         item1="U",
+                         it1num="X")
+
+# Get the value of the displacements in Y-direction.
+disp_y = mapdl.get_value(entity="NODE",
+                         entnum=2,
+                         item1="U",
+                         it1num="y")
+
+
+###############################################################################
+# Check Results
+# ~~~~~~~~~~~~~
+# Finally we have the results of the strain energy and
+# displacements in :math:`X` and :math:`Y` directions, which can be compared with
+# expected target values:
+#
+# - Strain energy of the system :math:`U_{\mathrm{(energy)}} = 24.01\,N-cm`.
+# - Displacement in X-direction :math:`U_x = 8.631\,cm`.
+# - Displacement in Y-direction :math:`U_y = 4.533\,cm`.
+#
+# For better representation of the results we can use ``pandas`` dataframe
+# with following settings below:
+
+# Define the names of the rows.
+row_names = ["Strain Energy, N-cm",
+             "Deflection-x , cm",
+             "Deflection-y , cm"]
+
+# Define the names of the columns.
+col_names = ['Target',
+             'Mechanical APDL',
+             'RATIO']
+
+# Define the values of the target results.
+target_res = np.asarray([24.01, 8.631, 4.533])
+
+# Create an array with outputs of the simulations.
+simulation_res = np.asarray([strain_energy, disp_x, disp_y])
+
+# Identifying and filling corresponding columns.
+main_columns = {
+    "Target": target_res,
+    "Mechanical APDL": simulation_res,
+    "Ratio": list(np.divide(simulation_res, target_res))
+}
+
+# Create and fill the output dataframe with pandas.
+df2 = pd.DataFrame(main_columns, index=row_names)
+
```

Comment:
from ansys.mapdl.core import data_frame_formatter

df = data_frame_formatter(df, font_size=20, background_color='black', format_str="{:.2f}")

Reply:
I would go leave it, and go with:

https://github.com/pyansys/pyansys-sphinx-theme/pull/17

which should be default style from now on.

Reply:
Hi @akaszynski  and @germa89 
as we have been talking with Alex 
I will create and finish my data frame with oop implementation. 

I would like to ask you @germa89  to check this PR with this styling of pandas dataframe.

When I finish PR #848 I will create a new pr wher e i will change all data frames for vms.

thank you in advance.